openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-scriptrunner/src/tools/ScriptRunner/ScriptRunner.vue

```diff
@@ -1358,6 +1358,12 @@ export default {
     },
     async executeText(text, breakpoints = []) {
       let extension = this.fullFilename.split('.').pop()
+      if (extension.includes(' *')) {
+        extension = extension.split(' *')[0]
+      }
+      if (extension !== 'rb' && extension !== 'py') {
+        extension = 'rb' // Still default to Ruby if we can't determine
+      }
```

Comment:
After testing I think the real issue was we were pushing files with `.rb *` in the filename and then the file could not be found.

Reply:
Note that if the file completes early and we hit a route like `/tools/scriptrunner/99` we already open up the Completed Scripts pane and show that. Now that we have the ID in the completed scripts it should be easy to make the connection.